jupyter-pystata/resources/Stata.ipynb

```diff
@@ -6,29 +6,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
+    "import shutil\n",
+    "if (os.path.isfile(f\"/home/{os.environ['NB_USER']}/pd/stata.lic\")\n",
+    "    and not os.path.isfile(\"/usr/local/stata17/stata.lic\")):\n",
+    "    shutil.copyfile(f\"/home/{os.environ['NB_USER']}/pd/stata.lic\", \"/usr/local/stata17/stata.lic\")\n",
+    "\n",
```

Comment:
If the Stata license is copied in the first cell of the notebook then the script for `copy_license_on_startup.py` would not be required.